src/main/java/me/rigamortis/seppuku/impl/module/movement/ElytraFlyModule.java

```diff
@@ -167,9 +167,9 @@ public void onWalkingUpdate(EventUpdateWalkingPlayer event) {
                             mc.player.motionX = 0;
                             mc.player.motionZ = 0;
                             if (mc.player.movementInput.jump) {
```

Comment:
`upspd` and `downspd` aren't values, if you want to add them you can, but leaving them as half the speed is fine.

Also `mc.player.motionY = -this.idlefall.getValue();` seemed to have been previously added without a corresponding value, please replace `-this.idlefall.getValue();` with '0' if you are working on elytrafly. This isn't the right place to be setting the fall speed of the player, that has to be done in a patch / event for  `EntityLivingBase.travel()`

Reply:
Ok, i understand everything but the:

  if (mc.player.movementInput.jump) {
                            mc.player.motionY = this.upspd.getValue();
                            } else if (mc.player.movementInput.sneak) {
                            mc.player.motionY = -this.downspd.getValue();
                            }

What values are supposed to be there instead of upspd and downspd

Reply:
I'm now realising that this was changed by an earlier pr. It used to be `mc.player.motionY = this.speed.getValue() / 2;` instead of `mc.player.motionY = this.upspd.getValue();`. They probably forgot to add those values before opening a pr.

So you could either add those values or you could change it back to `mc.player.motionY = this.speed.getValue() / 2;`  and `mc.player.motionY = -this.speed.getValue() / 2;`.

Reply:
Alright, i just pushed and fixed all the problems you mentioned.

Reply:
> I'm now realising that this was changed by an earlier pr. It used to be `mc.player.motionY = this.speed.getValue() / 2;` instead of `mc.player.motionY = this.upspd.getValue();`. They probably forgot to add those values before opening a pr.

> Also `mc.player.motionY = -this.idlefall.getValue();` seemed to have been previously added without a corresponding value

Yeah, that was my fault when copying the diff between branches, i will open a PR to fix it.